task_generator/task_generator/constants.py

```diff
@@ -10,15 +10,17 @@
 from task_generator.shared import Namespace, rosparam_get
 import dynamic_reconfigure.client
 
+
 class Constants:
 
     DEFAULT_PEDESTRIAN_MODEL = "actor1"
 
-    TASK_GENERATOR_SERVER_NODE = Namespace("task_generator_server")
+    TASK_GENERATOR_SERVER_NODE = Namespace("/task_generator_server")
```

Comment:
does this work with multi env training? @tuananhroman @MaximilianSchreff

Reply:
Tested it with both versions and also testing mode and non-testing mode. The task_generator_server node was never launched in a namespace. I.e. the services always have the global name /task_generator_server/...